src/content/docs/2/reference/cli.mdx

```diff
@@ -0,0 +1,508 @@
+---
+title: Command Line Interface (CLI)
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+The Tauri command line interface (CLI) is the way to interact with Tauri throughout the development lifecycle.
+
+You can add the Tauri CLI to your current project using your package manager of choice:
+
+<Tabs>
+	<TabItem label="npm">
+
+```sh
+npm install --save-dev @tauri-apps/cli@next
+```
+
+  </TabItem>
+  	<TabItem label="Yarn">
+
+```sh
+yarn add -D @tauri-apps/cli@next
+```
+
+  </TabItem>
+  	<TabItem label="pnpm">
+
+```sh
+pnpm add -D @tauri-apps/cli@next
+```
+
+  </TabItem>
+  	<TabItem label="Cargo">
+
+```sh
+cargo install tauri-cli@next
```

Comment:
This is my syntax, is there an equivalent to `next` in Cargo? I only see the alpha numbers in the blog posts so far.

Reply:
nope. only stuff like `cargo install tauri-cli --version 2.0.0-alpha` exists but no extra tags like in npm world afaik

Reply:
Alright, so then this just might be a necessary evil until we switch 2.0 over to release then. I'll just add one of those fancy comments to help us remember when that release time comes.